magma/passes/insert_coreir_wires.py

```diff
@@ -108,7 +110,8 @@ def _insert_wire(self, value, definition):
             return  # undriven value, skip wire insertion
         driver = value.value()
 
-        while (driver is not None and driver.name.anon() and
+        while (driver is not None and
```

Comment:
just noting that i see this same condition in `interface.py` above, and again below in this file. maybe a common function